lib/application/widgets/theme/contributors_button.dart

```diff
@@ -0,0 +1,149 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_hooks/flutter_hooks.dart';
+import 'package:layoutr/layoutr.dart';
+
+import 'package:memo/application/theme/theme_controller.dart';
+import 'package:memo/application/widgets/theme/link.dart';
+import 'package:memo/domain/models/contributor.dart';
+import 'package:memo/application/constants/dimensions.dart' as dimens;
+import 'package:memo/application/constants/strings.dart' as strings;
+import 'package:memo/application/constants/images.dart' as images;
+import 'package:memo/application/utils/bottom_sheet.dart';
+
+/// An alternative to the [SingleContributorButton]
+/// when there's more than one contributor to a collection
+///
+/// This button meets the need of a layout to multiple contributors
+/// wraping them togheter.
```

Comment:
wraping -> wrapping
togheter -> together

We also use 120 characters as the max line length, so you could increase this doc line length to respect this 120 characters

Reply:
Also, a suggestion to improve your class docs is to follow the same pattern that Flutter does:

```dart
/// A brief description
/// (empty line)
/// Other important information from the Widget
```

As an example, I would reformat these docs like

```dart
/// A visual opinionated button that shows multiple contributors from a collection 
///
/// An alternative to the [SingleContributorButton] when there's more than one contributor for the collection
/// When tapped shows a modal bottom sheet with a list of [SingleContributorButton]'s for each contributor.
```